tools/rosetta/cmd/rosetta/main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"os"
+
+	runtimev1alpha1 "cosmossdk.io/api/cosmos/app/runtime/v1alpha1"
+	appv1alpha1 "cosmossdk.io/api/cosmos/app/v1alpha1"
+	"cosmossdk.io/core/appconfig"
+	"cosmossdk.io/depinject"
+	rosettaCmd "cosmossdk.io/tools/rosetta/cmd"
+	"cosmossdk.io/tools/rosetta/lib/logger"
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	_ "github.com/cosmos/cosmos-sdk/runtime"
+)
+
+var Config = appconfig.Compose(&appv1alpha1.Config{
+	Modules: []*appv1alpha1.ModuleConfig{
+		{
+			Name: "runtime",
+			Config: appconfig.WrapAny(&runtimev1alpha1.Module{
+				AppName: "rosetta",
+			}),
+		},
+	},
+})
+
+func main() {
+	var (
+		logger            = logger.NewLogger()
+		cdc               codec.Codec
+		interfaceRegistry codectypes.InterfaceRegistry
+	)
+
+	if err := depinject.Inject(Config, &cdc, &interfaceRegistry); err != nil {
+		panic(err)
+	}
```

Comment:
I don't quite follow why this is needed

Reply:
Just using the latest cool things :) but removing then :D

Reply:
changed

Reply:
ah sorry, I wasn't following so was curious. but it does look simpler now. Does it need any other types that we may not have in the sdk?